README.md

```diff
@@ -139,6 +139,58 @@ example:
 
 A SessionID remains valid for 24 hours.
 
+#### Logout from Session
+
+New sessionID will be generated for each task run from Ansible collection SDK if a sessionID is not passed as param to task. And the session will be logged out once task is done.
```

Comment:
Ansible SDK handles OneView session in two different way
1. By default OneView session will be logged out when each task run from Ansible collection SDK. In this case always new session gets created whenever any task invoked from the SDK
2. One can run multiple tasks using same session. In this case first one needs to use task to get a session id and then use the same session id for all the subsequent tasks. At the end, logout task need to be invoked to delete that specific session

Reply:
done